ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_CppSamples/ArcGISRuntimeSDKQt_CppSamples.pro

```diff
@@ -196,7 +187,32 @@ SOURCES += \
     $$COMMONVIEWER/SearchFilterSimpleKeywordCriteria.cpp \
     $$COMMONVIEWER/SourceCode.cpp \
     $$COMMONVIEWER/SourceCodeListModel.cpp \
-    $$COMMONVIEWER/mainSample.cpp
+    $$COMMONVIEWER/mainSample.cpp \
+    $$files(zlib-ng/*.c) \
```

Comment:
files should use PWD or `$$COMMONVIEWER` or else the build could fail depending on the directory you're in when you run the build. It's important the build can be run from anywhere.

Reply:
Fixed and moved into it's own pri.